CHANGES.md

```diff
@@ -14,6 +14,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Advanced usage notebook][1810261633] now runs on Google Colab
 - [MNIST with scikit-learn and skorch][1811011230] now runs on Google Colab
 - Better user-facing messages when module or optimizer are re-initialized
+- Added the ability to register "virtual" parameters on the network to mark
+  them safe for setting without re-initialization. This is useful for specific
```

Comment:
Maybe add that this feature is experimental and a "public" API for user-defined parameters will be added later?

I would also split this message:
1) Add this experimental feature...
2) Setting lr, momentum etc. no longer resets optimizer.

That fact that the two are related does not matter that much from the user's point of view.